app/pages/sidedrawer.xml

```diff
@@ -14,8 +16,38 @@
       </ActionBar.actionItems>
     </ActionBar>
   </Page.actionBar>
-  <StackLayout class="demo-full">
-    <Label text="TO DO :)" textWrap="true" />
-     
-  </StackLayout>
+
+  <drawer:RadSideDrawer id="sideDrawer" drawerLocation="Left">
+      <drawer:RadSideDrawer.mainContent android:showOverNavigation="false">
+      <StackLayout cssClass="mainContent">
+            <Label text="Side Drawer Example" />
+      </StackLayout>
+    </drawer:RadSideDrawer.mainContent>
+    <drawer:RadSideDrawer.drawerContent >
+      <StackLayout class="sidedrawer-left">
+        <StackLayout class="sidedrawer-header">
+            <Image src="" stretch ="none" class="sidedrawer-header-image"/>
+            <Label text="name@gmail.com" textWrap="true" class="sidedrawer-header-brand" />
+        </StackLayout>
+          <StackLayout class="sidedrawer-content">
+          <StackLayout class="sidedrawer-list-item" >
+            <Label text="{{'fa-inbox' | fonticon }}" class="fa sidedrawer-list-item-icon" /><Button class="sidedrawer-list-item-text" text="Inbox" tap="" />
+          </StackLayout>
+          <StackLayout class="sidedrawer-list-item sidedrawer-list-item-selected">
```

Comment:
I like this classing structure @TheOriginalJosh nice work. I think on selected items, I'm on the fence with something like `sidedrawer-list-item-selected`.... I feel like since these can get length, I'm more for a simple `active` class which would always be scoped to that item so:
```
.sidedrawer-list-item {

  &.active {
     // etc.
```
`<StackLayout class="sidedrawer-list-item active">`

Idk, @tjvantoll any thoughts?